entry_not_found_error.go

```diff
@@ -3,4 +3,4 @@ package bigcache
 import "errors"
 
 // ErrEntryNotFound is an error type struct which is returned when entry was not found for provided key
-var ErrEntryNotFound = errors.New("Entry not found")
+var ErrEntryNotFound = errors.New("entry not found")
```

Comment:
https://github.com/allegro/bigcache/pull/168/files#r327431963

Reply:
Same here, changing the api.